app/Http/Livewire/Select.php

```diff
@@ -8,23 +8,20 @@
 
 class Select extends Component
 {
-    public $country;  //wire:model is not work
+    public $countryc;
     public $areas = [];
     public $area;   
     
     public function render()
     {
-        if(!empty($this->country)) {
-            $this->areas = Area::where('povince_id', $this->country)->get();
+        if(!empty($this->countryc)) {
+            $this->areas = Area::where('province_id', $this->countryc)->get();
         }
 
         $countries = Province::orderBy('name')->get();
```

Comment:
You can use `pluck` method to get `id` and `name` + sort.

Reply:
I can't use pluck because error id.